client/src/api/v1/cards/index.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+const api = {
+  v1: {
+    cards: async (responseHandler, errorHandler) => {
+      let response
+      try {
+        response = await axios.get('http://localhost:5000/api/v1/cards')
+        responseHandler(response)
+      } catch (error) {
+        errorHandler(error)
+      }
+    },
+  },
+}
+
+export default api
```

Comment:
@DDKatch I do like it 👍 

The only thing here - such requests should be placed near a container file, so:
```
AppContainer/
  index.js
  requests.js (or similar) P.S. there is a community name for such files, you may want to ask in #dev channel
```

And all requests should be described in this file
That's needed for keeping module system